src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/timeline/TimelineEntityMapper.kt

```diff
@@ -1,14 +1,17 @@
 package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.mapper.timeline
 
-import org.mapstruct.Mapper
+import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.domain.timeline.Timeline
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.timeline.TimelineEntity
 
-@Mapper(
-  uses = [
-    TimelineEventEntityMapper::class,
-  ],
-)
-interface TimelineEntityMapper {
-  fun fromEntityToDomain(persisted: TimelineEntity): Timeline
+@Component
+class TimelineEntityMapper(private val timelineEventEntityMapper: TimelineEventEntityMapper) {
```

Comment:
Given that I had to touch the mapper, it made sense to change it from mapstruct to a regular class